msi/jenkins.wxs

```diff
@@ -43,28 +43,47 @@
     <Upgrade Id="${MSI_PRODUCTCODE}">
       <UpgradeVersion Minimum="0.0.0" Maximum="99.99.99" Property="PREVIOUSVERSIONINSTALLED" />
     </Upgrade>
+
+    <CustomAction Id="BackupJenkinsXmlFile" Script="vbscript">
+      <![CDATA[
+        jenkinsDirPath = session.Property("JENKINSDIR")
+        Set fsObj = CreateObject("Scripting.FileSystemObject")
+
+        If fsObj.FolderExists(jenkinsDirPath) Then
+            jenkinsXmlPath = jenkinsDirPath + "jenkins.xml"
+
+            If fsObj.FileExists(jenkinsXmlPath) Then
+                Set shellObj = CreateObject("WScript.Shell")
+                cmdToExecute = "cmd.exe /C copy """ + jenkinsXmlPath + """ """ + jenkinsXmlPath + ".backup"" /Y"
```

Comment:
`cmd.exe` **may** be prohibited on newest Windows versions by admins, powershell is an old-new tend. I would advice against new CMD calls when it is not required. Hence 🐛 in this case